source/stylesheets/refills/_phone.scss

```diff
@@ -0,0 +1,42 @@
+.phone {
+  $ratio: 1.6;
+  $inset-shadow: inset 0 1px 4px transparentize(black, 0.8);
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.